examples/server/tests/features/environment.py

```diff
@@ -67,11 +76,11 @@ def server_kill_hard(context):
     path = context.server_path
 
     print(f"Server dangling exits, hard killing force {pid}={path}...\n")
-    if os.name == 'nt':
-        process = subprocess.check_output(['taskkill', '/F', '/pid', str(pid)]).decode()
-        print(process)
-    else:
-        os.kill(-pid, signal.SIGKILL)
+    try:
+        psutil.Process(pid).kill()
+    except psutil.NoSuchProcess:
+        return False
+    return True
```

Comment:
@phymbert This calls `TerminateProcess(handle, SIGTERM)` on Windows and `os.kill(pid, SIGKILL)` on Unix. `os.kill(pid, signal.SIGTERM)` also calls TerminateProcess on Windows. psutil really seems like overkill here.

Reply:
Again,feel free to issue a PR